agave_app/renderer.h

```diff
@@ -30,6 +29,58 @@ class IRenderWindow;
 class RenderSettings;
 class Scene;
 
+// serialized so permanent?
+enum eRenderDurationType
+{
+  TIME = 0,
+  SAMPLES = 1
+};
+
+struct RenderDuration
+{
+  int samples = 1;
+  int duration = 0; // in seconds
+  eRenderDurationType durationType = SAMPLES;
+};
+
+struct CaptureSettings
+{
+  std::string outputDir;
+  std::string filenamePrefix;
+  int width;
+  int height;
+  RenderDuration renderDuration;
+  int startTime;
+  int endTime;
+
+  CaptureSettings()
+  {
+    // defaults!
+    QString docs = QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation);
+    outputDir = docs.toStdString();
+
+    filenamePrefix = "frame";
+    width = 0;
+    height = 0;
+    renderDuration.duration = 10;
+    renderDuration.samples = 32;
+    renderDuration.durationType = SAMPLES;
+    startTime = 0;
+    endTime = 0;
+  }
+
+  CaptureSettings(const CaptureSettings& other)
+  {
+    outputDir = other.outputDir;
+    filenamePrefix = other.filenamePrefix;
+    width = other.width;
+    height = other.height;
+    renderDuration = other.renderDuration;
+    startTime = other.startTime;
+    endTime = other.endTime;
+  }
+};
+
```

Comment:
all the above changes were mainly to move these data structures into this class when they were previously in RenderDialog.